en-US/win10/samples/NodejsCylon.md

```diff
@@ -66,19 +66,6 @@ Cylon.robot({
 * Run `npm install cylon cylon-firmata cylon-gpio cylon-i2c`
 
 
-### Get Serialport
```

Comment:
Although 'get serialport' instructions are no longer needed I should replace it with a step to build serialport for arm which is not obvious. Will make the update.